modules/core/core-api/src/main/java/com/enonic/xp/content/UpdateContentParams.java

```diff
@@ -26,7 +27,7 @@
 
     private boolean requireValid;
 
-    private boolean inherited = false;
+    private Set<ContentInheritType> inherit = Set.of();
```

Comment:
It differs from CreateContentParams.
There inherit is an immutable final field set from a builder.